streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/ClientState.java

```diff
@@ -17,40 +17,34 @@
 
 package org.apache.kafka.streams.processor.internals.assignment;
 
+
 import java.util.HashSet;
 import java.util.Set;
 
 public class ClientState<T> {
+    private final Set<T> activeTasks;
+    private final Set<T> standbyTasks;
+    private final Set<T> assignedTasks;
+    private final Set<T> prevActiveTasks;
+    private final Set<T> prevAssignedTasks;
 
-    final static double COST_ACTIVE = 0.1;
-    final static double COST_STANDBY  = 0.2;
-    final static double COST_LOAD = 0.5;
-
-    public final Set<T> activeTasks;
-    public final Set<T> standbyTasks;
-    public final Set<T> assignedTasks;
-    public final Set<T> prevActiveTasks;
-    public final Set<T> prevAssignedTasks;
-
-    public double capacity;
-    public double cost;
+    private int capacity;
 
     public ClientState() {
-        this(0d);
+        this(0);
     }
 
-    public ClientState(double capacity) {
+    ClientState(int capacity) {
         this(new HashSet<T>(), new HashSet<T>(), new HashSet<T>(), new HashSet<T>(), new HashSet<T>(), capacity);
     }
 
-    private ClientState(Set<T> activeTasks, Set<T> standbyTasks, Set<T> assignedTasks, Set<T> prevActiveTasks, Set<T> prevAssignedTasks, double capacity) {
+    private ClientState(Set<T> activeTasks, Set<T> standbyTasks, Set<T> assignedTasks, Set<T> prevActiveTasks, Set<T> prevAssignedTasks, int capacity) {
```

Comment:
The private constructor is there for the `copy` method. I'm going to leave it as is.